be/src/runtime/fragment_mgr.h

```diff
@@ -58,7 +58,7 @@ class FragmentMgr : public RestMonitorIface {
     using FinishCallback = std::function<void(PlanFragmentExecutor*)>;
 
     FragmentMgr(ExecEnv* exec_env);
-    virtual ~FragmentMgr();
+    virtual ~FragmentMgr();//why virtual dtor?
```

Comment:
what does this comment mean?

Reply:
just delete the empty implement of virtual ~FragmentMgr，you should implement dtor if op= and copy ctor are necessary also. otherwise not implement accord the 3 rules

Reply:
```suggestion
    virtual ~FragmentMgr(); // why virtual dtor?
```